pkg/splunk/enterprise/indexercluster.go

```diff
@@ -573,7 +573,7 @@ func (mgr *indexerClusterPodManager) getClusterManagerClient(ctx context.Context
 	podName := fmt.Sprintf(splcommon.TestClusterManagerID, managerIdxcName, "0")
 	adminPwd, err := splutil.GetSpecificSecretTokenFromPod(ctx, mgr.c, podName, mgr.cr.GetNamespace(), "password")
 	if err != nil {
-		scopedLog.Error(err, "Couldn't retrieve the admin password from pod %v", err.Error())
+		scopedLog.Error(err, fmt.Sprintf("Couldn't retrieve the admin password from pod %v", err.Error()))
```

Comment:
Same here.  Just remove the `%v` and `err.Error()`

Reply:
I think this one was intended to be `podName`. Fixed in the latest commit.